src/results/aggregator.rs

```diff
@@ -73,10 +73,12 @@ pub async fn aggregate(
     config: &Config,
     upstream_search_engines: &[EngineHandler],
     safe_search: u8,
+    tcp_connection_keepalive: u8,
```

Comment:
```suggestion
```

I would suggest not adding this function parameter as we are already passing the `config` struct which will contain the parsed config option, so we can use that directly in the `.tcp_keepalive` option of the `ClientBuilder` struct. I hope you got my point :slightly_smiling_face: .